fdk/http/event_handler.py

```diff
@@ -0,0 +1,57 @@
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import io
+
+from fdk import context
+from fdk import constants
+from fdk import log
+from fdk import response
+from fdk import runner
+
+from fdk.http import routine
+
+
+def event_handle(connection, handle_code):
+    async def pure_handler(request_reader, response_writer):
+        log.log("in pure_handler")
```

Comment:
rm `log.log` ? few in here, before finalizing patch at least

Reply:
that's for the FDK debug, Python FDK provides an ability to trace the code execution for the very beginning.
as a user you can enable that by setting `FDK_DEBUG=1` as an app/fn config.

Reply:
there are bunch of `log.log(...)` in code and they are there for a reason, not for dummy prints.